### DISCUSSION TITLE
Way to check purge content setting? 🤔

### DISCUSSION THREAD

**Discussion by carltongibson** [tailwindlabs/tailwindcss#5055] [Help] [answered]
I'm looking for a quick way to verify matches for the glob patterns to the `purge` `content` setting. 

Say I have lots of different paths, it's not always clear that the pattern is what you want. 
A rough example: 

```javascript
purge: { 
  content: [
    "./src/**/*.js",
    "../**/src/**/*.html",
    ...
    "./src/**/**/*.html" 
]},
```

Matching different file types from parent and sibling and child directories. ... 

Is there a way (or how can one) for instance print out all files that match, as a way of making sure your patterns are correct? 

Thanks! 🏅

**Comment by kunlongxu** [comment]
Do you know this issue #5056 ?

**Reply by carltongibson** [reply]
That looks related but not identical. 

I'd like to enumerate all files that are matched by the purge setting.

**Comment by simonswiss** [comment, marked as answer]
Hey!

Not _exactly_ what you're asking for maybe, but I found `globitor` to be pretty useful:

https://github.com/ranyitz/globitor

It's a nice wrapped around `globby`. Basically, you can run `npx globitor` in your project root, and then test a glob pattern to see what files it matches.

![image](https://user-images.githubusercontent.com/11733036/111021463-1778d680-83d5-11eb-8f75-5e16419f1277.gif)

I _don't think_ it allows to have an array of globs, unfortunately, but you can test them one by one.

There are also online tools - I like this one:

https://globster.xyz/

You can paste a list of your directory tree paths, and then test the glob entries (only one at a time, unfortunately, it appears).

I looked for a vscode extension that would let you test that, as it would be pretty neat. Didn't find any!

Hope it helps 👍

**Reply by carltongibson** [reply]
@simonswiss Thanks for that. It looks exactly what I need. I will have a play! 🎁
